hannah/nas/functional_operators/operators.py

```diff
@@ -1,30 +1,56 @@
-from abc import ABC, abstractmethod
+#
+# Copyright (c) 2023 Hannah contributors.
+#
+# This file is part of hannah.
+# See https://github.com/ekut-es/hannah for further info.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
 import sys
-from typing import Any
+from abc import ABC, abstractmethod
 from copy import deepcopy
+from typing import Any
 
 import torch
-import torch.nn.functional as F
 import torch.fx as fx
+import torch.nn.functional as F
+
+from hannah.models.functional_net_test.expressions import padding_expression
 from hannah.nas.core.parametrized import is_parametrized
+
 # from hannah.nas.functional_operators.fx_wrapped_functions import conv2d_forward, linear_forward
 from hannah.nas.functional_operators.lazy import lazy
 from hannah.nas.functional_operators.op import Choice, Op, Tensor
```

Comment:
Shouldn't this be `ChoiceOp` instead of `Choice`?

Otherwise the following error occurs:

```
Traceback:
/usr/lib/python3.8/importlib/__init__.py:127: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
hannah/nas/test/test_target_desc_markdown.py:19: in <module>
    from hannah.nas.hardware_description.backend import MarkdownBackend
hannah/nas/hardware_description/backend/__init__.py:27: in <module>
    from .tvm import TVMBackend
hannah/nas/hardware_description/backend/tvm.py:21: in <module>
    from ...functional_operators.operators import ChoiceOp, Op, Tensor
E   ImportError: cannot import name 'ChoiceOp' from 'hannah.nas.functional_operators.operators' (/var/tmp/ga87puy/hannah_new/hannah/nas/functional_operators/operators.py)
```